wiki/Modding_Guides/Getting_Your_Map_Modded/en.md

```diff
@@ -1,11 +1,14 @@
-In order for your map to become [ranked](ranked_beatmap "wikilink"), it must get [modded](modding "wikilink"). There are multiple ways to accomplish this task.
+Getting Your Map Modded
+===========================
+
+n order for your map to become [ranked](/wiki/Beatmaps), it must get [modded](/wiki/Modding). There are multiple ways to accomplish this task.
```

Comment:
`In order for your map to become [ranked](/wiki/Beatmaps), it must get [modded](/wiki/Modding). There are multiple ways to accomplish this task.`

Missed the starting `I`